### PR TITLE
fix: replace netlify:edge identifier with full URL

### PR DESCRIPTION
**Pull request by khendrikse**
### Summary

This replaces `netlify:edge` with `https://edge.netlify.com`.

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal
Part of https://github.com/netlify/pillar-runtime/issues/322.

cute pug playing a guitar

### Standard checks:

- [ ] Check the Deploy Preview's Demo site for your PR's functionality
- [ ] Add docs when necessary

---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.

### CODE REVIEW

**Review by eduardoboucas — APPROVED** [review]
Looks good from my side, but let's wait for a 👍🏻 from the frameworks peeps.

File: plugin/src/templates/edge/ipx.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Accepts } from 'https://deno.land/x/accepts@2.1.1/mod.ts'
-import type { Context } from 'netlify:edge'
+import type { Context } from 'https://edge.netlify.com'
```
Comment (nickytonline) [inline plugin/src/templates/edge/ipx.ts:2]: For folks that are using the current plugin, will `netlify:edge` will still work?

Comment (khendrikse) [reply plugin/src/templates/edge/ipx.ts:2]: Let me verify!

Comment (khendrikse) [reply plugin/src/templates/edge/ipx.ts:2]: @nickytonline yes it should still work :)

**Review by ericapisani — APPROVED** [review]
LGTM 🚀 

Thanks for making this change, will be happy to have to do less IDE wrangling to get this to compile going forward 😄

**Comment by nickytonline** [comment]
Just looking into why a bunch of checks are failing.

**Review by nickytonline — APPROVED** [review]
Alright, I got all the checks to pass finally! 🚀

**Comment by khendrikse** [comment]
> Alright, I got all the checks to pass finally! 🚀

Thanks for looking at that @nickytonline 🙏

**Pull request merged**